### PR TITLE
Update workflow to use GitHub ssh

### LINKED ISSUE #2302: Add SSH Key and SSH Cloning to the getting-started.md docs
### Description of the task
Since github is forcing people to stop using username and passwords, SSH is now one of the only good options, so this should be part of the guide and not merely a suggestion. `git clone` should also be with the ssh link so ssh keys work properly

### Acceptance criteria

- [ ] update getting-started.md docs as described

### PR DESCRIPTION
**Pull request by Joshuayellowley**
### Description

Updated a few links that changed instructions to using github with SSH in the getting_started.md file.

### Testing Done

N/A

### Resolved Issues

resolves #2302 

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [x] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

**Review by itsarune — DISMISSED** [review]

File: docs/getting-started.md [hunk 1]
```diff
@@ -63,10 +63,10 @@ You can use Ubuntu 20.04 LTS inside Windows through Windows Subsystem for Linux,
 4. Click the `Fork` button in the top-right to fork the repository ([click here to learn about Forks](https://help.github.com/en/articles/fork-a-repo))
    1. Click on your user when prompted
    2. You should be automatically redirected to your new fork
-5. Clone your fork of the repository (you can put it wherever you want)
-   1.  Eg. `git clone https://github.com/<your-username>/Software.git`
-      1. You can find this link under the green `Clone or Download` button on the main page of the Software repository
-   2. We recommend cloning with SSH if you don't like typing your username and password all the time. Instructions can be found [here](https://help.github.com/articles/connecting-to-github-with-ssh/).
+5. Clone your fork of the repository. As GitHub is forcing users to stop using usernames and passwords, we will use the SSH link. (you can put it wherever you want)
+   1.  Eg. `git clone git@github.com:<your_username>/Software.git`
+      1. You can find this link under the green `Clone or Download` button on the main page of the Software repository, under the SSH tab
```
Comment (jonathanlew) [inline docs/getting-started.md]: you actually need to add an ssh key before the ssh tab will show up

**Review by jonathanlew — CHANGES_REQUESTED** [review]
please also change the instructions regarding the upstream remote to ssh

**Comment by jonathanlew** [comment]
please update the pr title

File: docs/getting-started.md [hunk 2]
```diff
@@ -63,10 +63,10 @@ You can use Ubuntu 20.04 LTS inside Windows through Windows Subsystem for Linux,
...
+   2. To connect to GitHub using SSH, if not setup prior, you will need to add an SSH key to your GitHub account. Instructions can be found [here](https://help.github.com/articles/connecting-to-github-with-ssh/).  For each computer you contribute to GitHub with, you will need an additional SSH Key pair linked to your account.
```
Comment (PhilD71) [inline docs/getting-started.md]: Might be better to direct here https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent. Saves a click

**Review by PhilD71 — COMMENTED** [review]
Added one comment - agree with Jon's comment about setting up the key first before cloning the repo.

File: docs/getting-started.md (hunk 2 above)
Comment (PhilD71) [reply docs/getting-started.md]: Also might be worth adding that if you've already cloned with an alternative method you need to update your remote to use ssh

File: docs/getting-started.md (hunk 1 above)
Comment (Joshuayellowley) [reply docs/getting-started.md]: added that ssh key must be added to github beforehand

**Review by jonathanlew — APPROVED** [review]
LGTM!

**Review by akhilveeraghanta — APPROVED** [review]

**Review by PhilD71 — APPROVED** [review]
Nice!

**Review by itsarune — APPROVED** [review]
awesome

**Pull request merged**